File: baekjoon/1차원 배열/10818_최소&최대.cpp
#include <iostream>
#include <stdio.h>
using namespace std;

int main(void) {
	/* #10818*/
	ios::sync_with_stdio(false);
	cin.tie(NULL);
	cout.tie(NULL);

	int N, temp, MAX = 1000000, MIN = -1000000;
    cin >> N;
	int* arr = new int[N];
	for (int i = 0; i < N; i++) {
		cin >> arr[i];
        if (MIN < arr[i]) {
			MIN = arr[i];
		}
		if (MAX > arr[i]) {
			MAX = arr[i];
		}
	}
	cout << MAX << " " << MIN;
	return 0;
}

File: baekjoon/1차원 배열/10818_최소&최대.java
// 10818_최소&최대.java

import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        
        int arr[] = new int[N];
        int min = 1000000, max = -1000000;
        
        for(int i = 0; i < N; i++){
            arr[i] = sc.nextInt();
            
            if(arr[i] > max) max = arr[i];
            if(arr[i] < min) min = arr[i];
        }
        
        System.out.println(min + " " + max);
        sc.close();
    }
}
